.travis.yml

```diff
@@ -188,6 +188,10 @@ before_scripts:
   -     curl -Ls -o debian.sh https://github.com/mate-desktop/mate-dev-scripts/raw/master/travis/debian.sh
   -     bash ./debian.sh
   - fi
+  - if [ ${DISTRO_NAME} == "debian" ];then
```

Comment:
```
  - fi
  - if [ ${DISTRO_NAME} == "debian" ];then
```

I think this two lines ^^^ can be removed, because previous condition works with debian too.

Reply:
Done.